<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

    <title>Network-Based Gene Set Analysis with Incomplete Network Information - Alexej Gossmann</title>

		<meta name="description" content="Tulane University Department of Mathematics.">
		<meta name="author" content="Alexej Gossmann">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="stylesheet" href="css/reveal.min.css">
		<link rel="stylesheet" href="css/theme/simple.css" id="theme">
		<link rel="stylesheet" href="css/theme/simple-custom.css">

		<!-- For syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- If the query includes 'print-pdf', include the PDF print sheet -->
		<script>
			if( window.location.search.match( /print-pdf/gi ) ) {
				var link = document.createElement( 'link' );
				link.rel = 'stylesheet';
				link.type = 'text/css';
				link.href = 'css/print/pdf.css';
				document.getElementsByTagName( 'head' )[0].appendChild( link );
			}
		</script>

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->

    <!-- MathJax interation, copied from http://cryptotoxin.github.io/coding/math-symbols-latex-and-markdown/ and others-->
    <script type="text/x-mathjax-config">
      MathJax.Hub.Config({
        tex2jax: {inlineMath: [["$","$"]]}
      });
      MathJax.Hub.Config({TeX: {Macros:{subscript:['_{#1}',1],superscript:['^{#1}',1]}}});
    </script> 
    <script type="text/javascript"
      src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
    </script>
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
        <section data-markdown>
          <script type="text/template">
## Network-Based Gene Set Analysis with Incomplete Network Information

### Alexej Gossmann
#### Tulane University
#### Dept. of Biostatistics and Bioinformatics Journal Club
### 2015/9/4
          </script>
        </section>


        <section>

          <section data-markdown>
            <script type="text/template">
## Background

### Motivation

* Test the significance of a pre-specified subnetwork (e.g. pathway).
* Incorporate the network structure.
* Consider changes in the network structure between different experimental conditions (e.g. case &mdash; control).
* Consider changes in the gene (protein, metabolite) expression.
            </script>
          </section>

          <section data-markdown>
            <script type="text/template">
## Background

### Gene Set Enrichment Analysis (Subramanian et. al., 2005, Efron and Tibshirani, 2007).

* Tests for the *joint* effect of biologically related groups of genes.
* Higher power and better interpretability than single gene analysis.
* Association measures (e.g. p-values) are computed for each gene separately. They are then combined into an *enrichment score* for each gene set without direct incorporation of correlations between genes.
* These methods do not incorporate network information.
            </script>
          </section>

        </section>


        <section>

          <section data-markdown>
            <script type="text/template">
## Network-Based Gene Set Analysis (NetGSA)

* Combines the ideas of gene set analysis methods, network-based single gene analysis, and linear mixed models.
* Assesses changes in gene expression as well as network structure of arbitrary subnetworks (e.g. pathways) between different experimental conditions (e.g. case &mdash; control).
* Provides a general framework for inference in complex experiments using the linear mixed models theory.
            </script>
          </section>

        </section>

        <section>

          <section data-markdown>
            <script type="text/template">
## Main idea

### Linear mixed model representation

Very simple graph:

![Simple network PNG](images/NetGSA/simple_network.png?raw=true)

$$\begin{eqnarray}
  X\subscript{1} &=& \gamma\subscript{1},\\\\\\
  X\subscript{2} &=& \rho\subscript{12} X\subscript{1} + \gamma\subscript{2} = \rho\subscript{12} \gamma\subscript{1} + \gamma\subscript{2},\\\\\\
  X\subscript{3} &=& \rho\subscript{23} X\subscript{2} + \gamma\subscript{3} = \rho\subscript{23}\rho\subscript{12}\gamma\subscript{1} + \rho\subscript{23} \gamma\subscript{2} + \gamma\subscript{3}.
\end{eqnarray}$$

where $\gamma\subscript{i} \sim N(\mu\subscript{i}, \sigma\subscript{\gamma}^2)$.

<br>
$$\Rightarrow X = \Lambda \gamma, \quad \gamma \sim N(\mu, \sigma\subscript{\gamma}^2 I).$$

            </script>
          </section>

          <section data-markdown>
            <script type="text/template">
## Main idea

### Linear mixed model representation

* Let $Y$ be the $i$th sample in the expression data.
* Assume that $Y = X + \varepsilon$, with $X$ the signal and $\varepsilon \sim N(0, \sigma\subscript{\varepsilon}^2 I)$ the noise.
* It follows that

  $$Y = \Lambda \gamma + \varepsilon,$$

  $$Y\sim N(\Lambda\mu, \sigma\subscript{\gamma}^2 \Lambda\Lambda^T + \sigma\subscript{\varepsilon}^2 I).$$
            </script>
          </section>

          <section data-markdown>
            <script type="text/template">
## Main idea

### Linear mixed model representation

* Multiple experimental conditions, e.g. case &mdash; control:

  $$\begin{eqnarray}
  Y^C &=& \Lambda^C \gamma^C + \varepsilon, \quad \gamma^C &\sim& N(\mu^C, \sigma\subscript{\gamma}^2 I), \\\\\\
  Y^T &=& \Lambda^T \gamma^T + \varepsilon, \quad \gamma^T &\sim& N(\mu^T, \sigma\subscript{\gamma}^2 I).
  \end{eqnarray}$$

* Let $\beta := ((\mu^C)^\prime, (\mu^T)^\prime)^\prime$, and rearrange $Y$, $\gamma$ and $\varepsilon$ into $np \times 1$ vectors, and redefine $\gamma$ such that $E(\gamma) = 0$. Then we obtain a linear mixed model:

  <br>
  $$Y = \Psi \beta + \Pi \gamma + \varepsilon,$$
  <br>

  where $\gamma \sim N(0, \sigma\subscript{\gamma}^2 I)$, $\Pi = I \otimes \Lambda \in \mathbb{R}^{np \times np}$, $\Psi = \chi \otimes \Lambda \in \mathbb{R}^{np \times 2p}$ ($\chi\in\mathbb{R}^{n\times 2}$ decodes membership in treatment or control group).
            </script>
          </section>

          <section data-markdown>
            <script type="text/template">
## Main idea

### Inference

![Simple network PNG](images/NetGSA/simple_subnetwork.png?raw=true)

* Use the LMM framework to test whether a subnetwork differs w.r.t. the network structure or gene expression levels between cases and controls.

* Any hypothesis test of the following form can be performed:

  $$H\subscript{0} : l^\prime \beta = 0 \quad \mathrm{vs.} \quad H\subscript{1} : l^\prime \beta \neq 0.$$

* Choose a contrast vector $l$ that includes the interaction between genes 2 and 3 and excludes gene 1.
            </script>
          </section>

        </section>

        <section>

          <section data-markdown>
            <script type="text/template">
## LMM general form

* $Y = \Psi \beta + \Pi \gamma + \varepsilon$, where $\beta$ and $\gamma$ are the fixed and respectively the random effects coefficients.

* $\gamma \sim N(0, \sigma\subscript{\gamma}^2 I)$, $\varepsilon \sim N(0, R(\theta\subscript{\varepsilon}))$, where *temporal correlations* may be incorporated through $R$. 

* Estimates of $\sigma\subscript{\gamma}^2$ and $\theta\subscript{\varepsilon}^2$ are obtained based on the REML criterion.

* MLE of $\beta$: 
  $$\hat{\beta} = (\Psi^\prime \hat{W}^{-1} \Psi)^{-1} \Psi^\prime \hat{W}^{-1} Y,$$ 
  where $W = \sigma\subscript{\gamma}^2 \Pi \Pi^\prime + R$.
            </script>
          </section>

          <section data-markdown>
            <script type="text/template">
## LMM general form

### Influence matrix

* The fixed and random effects design matrices $\Psi$ and $\Pi$ are determined from the *influence matrix* $\Lambda$.

* The influence matrix represents the effect of each gene on all of the other genes in the network.

* In general, $\Lambda = \sum\subscript{r=0}^\infty A^r$, where $A$ is the adjacency matrix of the network.

* In practice, one can compute 
  $$\Lambda = \lim\subscript{\zeta\to 0} (I - \mathcal{L}(\zeta))^{-1} = (I - \mathcal{L}(0))^+,$$
  where $(\cdot)^+$ denotes the Moore-Penrose pseudo-inverse, and
  $$\mathcal{L}(\zeta)\subscript{ij} = \frac{A\subscript{ij}}{(\sum\subscript{j=1}^p |A\subscript{ij}|)+\zeta}.$$
            </script>
          </section>

        </section>


        <section>

          <section data-markdown>
            <script type="text/template">
## Inference

* Any hypothesis about the fixed effects parameters of a LMM can be tested by considering tests of the following form:

  $$H\subscript{0} : l^\prime \beta = 0 \quad \mathrm{vs.} \quad H\subscript{1} : l^\prime \beta \neq 0,$$
  or
  $$H\subscript{0} : L \beta = 0 \quad \mathrm{vs.} \quad H\subscript{1} : L \beta \neq 0.$$

* The authors propose a $t$-test and an $F$-test whose degrees of freedom are approximated with the Satterthwaite method, using the test statistics

  $$T = \frac{l^\prime \hat{\beta}}{\sqrt{l\hat{C}l^\prime}},$$
  and
  $$F = \frac{\hat{\beta}^\prime L^\prime (L\hat{C}L^\prime)^{-1} L\hat{\beta}}{\mathrm{rank}(L)},$$
  where $C = (\Psi^\prime W^{-1} \Psi)^{-1}$.
            </script>
          </section>

          <section data-markdown>
            <script type="text/template">
## Inference

### Choice of the contrast vector

* The choice of $l$ in $H\subscript{0} : l^\prime \beta = 0$ is crucial.

* For any indicator vector $b$ determining a specific subnetwork, the vector $(b^\prime \Lambda \cdot b) \beta$ includes the effects of all nodes in $b$, but excludes the effects of nodes not in $b$ (here $\cdot$ denotes the Hadamard product).
            </script>
          </section>

        </section>


        <section>

          <section data-markdown>
            <script type="text/template">
            </script>
          </section>

        </section>

			</div>
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.min.js"></script>

		<script>

			// Full list of configuration options available here:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,

				theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
				transition: Reveal.getQueryHash().transition || 'linear', // default/cube/page/concave/zoom/linear/fade/none

				// Parallax scrolling
				// parallaxBackgroundImage: 'https://s3.amazonaws.com/hakim-static/reveal-js/reveal-parallax-1.jpg',
				// parallaxBackgroundSize: '2100px 900px',

				// Optional libraries used to extend on reveal.js
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
					{ src: 'plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } }
				]
			});

		</script>

	</body>
</html>
